src/Adapter/AbstractAdapter.php

```diff
@@ -179,16 +179,6 @@ public function setPageStatus($status, $pageId)
         throw new NotImplementedException('Provider does not support this feature.');
     }
 
-    /**
-     * {@inheritdoc}
-     *
-     * Checking access_token only works for oauth1 and oauth2, openid will overwrite this method.
-     */
-    public function isConnected()
```

Comment:
instead of removing this method - you need to make it as abstract. So `{@inheritdoc}` will work